website/docs/reference/datasources/querying-mysql.md

```diff
@@ -16,14 +16,21 @@ Appsmith supports MySQL versions 5.5, 5.6, 5.7, and 8.0.
 ### Configuration
 
 <figure>
-  <img src="/img/as-mysql-datasource-config.png" style= {{width:"700px", height:"auto"}} alt="MySQL Datasource configuration page"/>
+  <img src="/img/as-mysql-datasource-config.png" style= {{width:"100%", height:"auto"}} alt="MySQL Datasource configuration page"/>
   <figcaption align = "center"><i>MySQL Datasource configuration page</i></figcaption>
 </figure>
 
 :::tip
 If you want to connect to a local database, you can use a service like [ngrok](https://ngrok.com/) to expose it. For more information, see [How to connect to local database on Appsmith](/advanced-concepts/more/how-to-work-with-local-apis-on-appsmith).
 :::
 
+:::caution important
```

Comment:
Keep this callout above the image so that we don't to many callout clubbed together